api/v1alpha1/xlinecluster_types.go

```diff
@@ -47,6 +49,43 @@ type XlineClusterList struct {
 	Items           []XlineCluster `json:"items"`
 }
 
+type XlineArgs struct {
+	JaegerOffline               bool    `json:"jaeger-offline,omitempty"`
+	JaegerOnline                bool    `json:"jaeger-online,omitempty"`
+	JaegerLevel                 bool    `json:"jaeger-level,omitempty"`
+	ClientUseBackoff            bool    `json:"client-use-backoff,omitempty"`
+	AuthPrivateKey              *string `json:"auth-private-key,omitempty"`
+	AuthPublicKey               *string `json:"auth-public-key,omitempty"`
+	JaegerOutputDir             *string `json:"jaeger-output-dir,omitempty"`
+	LogFile                     *string `json:"log-file,omitempty"`
+	LogRotate                   *string `json:"log-rotate,omitempty"`
+	LogLevel                    *string `json:"log-level,omitempty"`
```

Comment:
It's better to use `kubebuilder:validation:Enum` to specify all the possible values for enum args.  FYI: https://book.kubebuilder.io/reference/markers/crd-validation